### PR TITLE
Migrated Datagrid Resize to Table Resize

### PR DESCRIPTION
**Pull request by David-Moreira**
Smooth transition to Table Resize leaving only the Resize Parameters.
Removed wwwroot from datagrid, for the time being. Datagrid no longer has required resources again.

### CODE REVIEW

**Comment by stsrki** [comment]
Nice, can make a note in the documentation on Datagrid page? And also on 094 release notes, in migration list?

**Comment by David-Moreira** [comment]
Sure.

File: Source/Extensions/Blazorise.DataGrid/DataGrid.razor [hunk 1]
```diff
@@ -8,7 +8,7 @@
     {
         <_DataGridPagination TItem="TItem" PaginationContext="@PaginationContext" PaginationTemplates="@PaginationTemplates" OnPaginationItemClick="@Paginate" New="@New" Edit="@Edit" Delete="@Delete" SelectedRow="@SelectedRow" ClearFilter="@ClearFilter" />
     }
-    <Table @ref="@tableRef" ElementId="@ElementId" Class="@ClassNames" Style="@Style" Margin="@Margin" Padding="@Padding" Striped="@Striped" Bordered="@Bordered" Borderless="@Borderless" Hoverable="@Hoverable" Narrow="@Narrow" Responsive="@Responsive">
+    <Table @ref="@tableRef" ElementId="@ElementId" Class="@ClassNames" Style="@Style" Margin="@Margin" Padding="@Padding" Striped="@Striped" Bordered="@Bordered" Borderless="@Borderless" Hoverable="@Hoverable" Narrow="@Narrow" Responsive="@Responsive" Resizable="@Resizable" ResizeMode="@(Enum.Parse<TableResizeMode>(ResizeMode.ToString("d")))">
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/DataGrid.razor:11]: Just remove DataGridResizeMode and use ResizeMode directly.

Comment (David-Moreira) [reply Source/Extensions/Blazorise.DataGrid/DataGrid.razor:11]: Fine, will update.

Comment (David-Moreira) [reply Source/Extensions/Blazorise.DataGrid/DataGrid.razor]: Updated. Will be a simple fix breaking change. I will put it in the docs.

**Comment by David-Moreira** [comment]
Done. You may go ahead and merge if everything looks good.

File: Source/Extensions/Blazorise.DataGrid/DataGrid.razor (hunk 1 above)
Comment (stsrki) [reply Source/Extensions/Blazorise.DataGrid/DataGrid.razor]: Looking good. I just did small formating fixes in markdown.

**Pull request merged**